pages/science/reactions.rst

```diff
@@ -344,6 +385,21 @@ Sticking reactions can be defined in the CTI format using the `stick` entry, or
 in the YAML format by specifying the rate constant in the reaction's
 :ref:`sticking-coefficient <sec-yaml-interface-reaction>` field.
 
+Surface Blowers-Masel Reactions
+-------------------------------
+
+Surface Blowers-Masel Reactions have the same Arrhenius-like rate expression described in
+:ref:`Surface Reactions<sec-surface>`, and the activation energy :math:`E_a` is determined
+as described in :ref:`Blowers-Masel Reactions<sec-Blowers-Masel>`.
+
+Surface Blowers-Masel reactions can be identified by the presence of surface spcecies and
+the defined :ref:`Blowers-Masel<sec-yaml-surface-Blowers-Masel>` reaction ``type`` in YAML
+format.
+
+Note that surface Blowers-Masel reactions also support all the :ref:`additional options <sec-yaml-interface-reaction>`
```

Comment:
Likewise here for the URL to the dev docs. Even though this exists in the stable docs, I think it'd be better to link to the dev docs so that the reader has everything they want on one page.